docs/spec2017-tutorial.md

```diff
@@ -0,0 +1,586 @@
+# Tutorial: Run SPEC CPU 2017 Benchmarks in Full System Mode with gem5art  
+
+## Introduction  
+In this tutorial, we will demonstrate how to utilize gem5art to run SPEC CPU 2017 benchmarks in gem5 full system mode. 
+The full example with all of the gem5art tutorials can be found [here](https://github.com/darchr/gem5art-experiments). 
+The scripts in this tutorial work with gem5art-* v0.3.1.  
+
+**Note**: This steps in this tutorial are mostly identical to that of [the SPEC 2006 tutorial](spec2006-tutorial.md). 
```

Comment:
Do you think it would be better to *not* replicate the text here and instead just say "Here's what needs to change"?

Reply:
I would expect someone to read SPEC 2017 tutorial without reading the other one. I'll point to the differences at the beginning of this tutorial.